paddle/operators/cross_entropy_op.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+#include "glog/logging.h"
+#include "paddle/framework/operator.h"
+
+namespace paddle {
+namespace operators {
+
+template <typename Place, typename T>
+class OnehotCrossEntropyOpKernel : public framework::OpKernel {
+public:
+  constexpr T LOG_THRESHOLD() const { return static_cast<T>(1e-20); }
```

Comment:
Would it be easier to have constant instead of a function here?

I tried the following program builds and runs:

```cpp
#include <iostream>

template <typename T>
struct Hello {
  const T kThreshold = static_cast<T>(1e-20);
};

int main() {
  Hello<float> h;
  std::cout << h.kThreshold << "\n";
  return 0;
}
```